### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by sungam3r**

### CODE REVIEW

File: samples/Samples.Server/Startup.cs [hunk 1]
```diff
@@ -10,6 +10,7 @@
 using GraphQL.Server.Ui.GraphiQL;
 using GraphQL.Server.Ui.Playground;
 using GraphQL.Server.Ui.Voyager;
+using GraphQL.SystemReactive;
```
[the remark this answers is not in the document]
Comment (Shane32) [reply samples/Samples.Server/Startup.cs:13]: I'm not sure yet.  We should discuss it.  Perhaps review a PR if it's easy to write one (looking at the api diff).  It's probably a good idea, so long as the extension methods relate to the library, like "AddSystemTextJson", which I think is already the case.

**Review by Shane32 — APPROVED** [review]

**Review by Shane32 — APPROVED** [review]

File: samples/Samples.Server/StartupWithRouting.cs [hunk 2]
```diff
@@ -42,7 +42,7 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddGraphQL(builder => builder
                 .AddServer(true)
-                .AddDocumentExecuter<SubscriptionDocumentExecuter>()
+                .AddSubscriptionExecutionStrategy()
```
Comment (Shane32) [inline samples/Samples.Server/StartupWithRouting.cs:45]: This does need to be updated...

Comment (sungam3r) [reply samples/Samples.Server/StartupWithRouting.cs:45]: Why?

Comment (sungam3r) [reply samples/Samples.Server/StartupWithRouting.cs:45]: Compiler shows warning, so I changed.

Comment (Shane32) [reply samples/Samples.Server/StartupWithRouting.cs:45]: Well, that's correct.  So why didn't we merge this PR?

Comment (sungam3r) [reply samples/Samples.Server/StartupWithRouting.cs:45]: #727

Comment (Shane32) [reply samples/Samples.Server/StartupWithRouting.cs:45]: Just to be clear, I'm agreeing with this change, not against it.  Just puzzled why the PR was closed without being merged.

Comment (sungam3r) [reply samples/Samples.Server/StartupWithRouting.cs:45]: I did not want to resolve conflicts.

Comment (Shane32) [reply samples/Samples.Server/StartupWithRouting.cs:45]: I already resolved all conflicts

**Pull request closed without merging**